templates/supermicro/form-data.json

```diff
@@ -0,0 +1,42 @@
+{
+  "form": {
+    "/supermicro": {
+      "templatePath": "/supermicro/index.html",
+      "isModal": true,
```

Comment:
We should consider making `isModal` configurable when calling `load_form`. Then we would need duplicate form datas. But this is out of scope here

Reply:
Yes
I agree. I created a ticket for [it](https://warthogs.atlassian.net/browse/WD-27754) but didn't get the time to implement.
Thanks for pointing